resources/js/processes/components/CreateProcessModal.vue

```diff
@@ -312,7 +312,7 @@ export default {
           } else {
             window.location =
               this.isAiGenerated
-                ? "/modeler/" + response.data.id + "?ai=true"
+                ? "/modeler/" + response.data.id + "?isAiGenerated=true"
```

Comment:
Please keep the variable `ai` instead of `isAiGenerated`, this change could be affect our LucidChart integration